src/ks-app/.angular-cli.json

```diff
@@ -20,7 +20,7 @@
       "prefix": "KS",
       "styles": [
         "../node_modules/clarity-icons/clarity-icons.min.css",
-        "../node_modules/clarity-ui/clarity-ui.min.css",
+        "../node_modules/clarity-ui/clarity-ui-dark.min.css",
```

Comment:
Is it possible to flip back to before dark theme for KS easily or have you removed things that make it look alright?

Reply:
I touched on this in the overview last week ... its one of the less than optimal dev items that I need to figure out. With how we are using webpack the best way to `flip` back and forth is to comment/uncomment the dark theme variables file while running the demo app in `_theme.dark.clarity.scss`

One idea I had is to split them out and export multiple configurations(https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations) so that both css files get added into the demo app and we can add a component into the app that toggles between the two stylesheets.